text/0000-edition-2021-and-beyond.md

```diff
@@ -0,0 +1,207 @@
+  - We will roll out an edition regardless of whether there are breaking changes.
+- Unlike Rust 2018, we will avoid using editions as a "deadline" to tie together high-priority projects.
+  - Instead, we embrace the train model, but editions are effectively a "somewhat bigger release", giving us an opportunity to give an overview of all the work that has landed over the previous three years.
+- We specify a cadence for Edition lints.
+  - "Edition idiom" lints for Edition N will warn for editions before N, and become "deny by default" in Edition N.
+  - Since it would be disruptive to introduce deny-by-default lints for Rust 2018 now, the Rust 2018 lints are repurposed into Rust 2021 Edition lints.
+- We specify a policy on reserving keywords and other prospective changes.
+  - In short, reserving keywords is allowed only as part of an active project group.
+
+# Motivation
+
+[motivation]: #motivation
+
+The plan for editions was laid out in [RFC 2052] and Rust had its first edition in 2018. This effort was in many ways a success but also resulted in some difficult lessons. As part of this year's roadmap, one of the major questions we identified was that we need to decide whether we are going to do more editions and -- if so -- how we are going to manage the process.
+
+[rfc 2052]: https://github.com/rust-lang/rfcs/blob/master/text/2052-epochs.md
+
+This RFC proposes various clarifications to the edition process going forward:
+
+- We will do new Rust editions on a regular, three-year cadence.
+  - We will roll out an edition regardless of whether there are breaking changes.
+- Unlike Rust 2018, we will avoid using editions as a "deadline" to tie together high-priority projects.
+  - Instead, we embrace the train model, but editions are effectively a "somewhat bigger release", giving us an opportunity to give an overview of all the work that has landed over the previous three years.
+- We specify a cadence for Edition lints.
+  - "Edition idiom" lints for Edition N will warn for editions before N, and become "deny by default" in Edition N.
+  - Since it would be disruptive to introduce deny-by-default lints for Rust 2018 now, the Rust 2018 lints are repurposed into Rust 2021 Edition lints.
+- We specify a policy on reserving keywords and other prospective changes.
+  - In short, reserving keywords is allowed only as part of an active project group.
+
+## Expected nature of editions to come
+
+We believe the Rust 2018 was somewhat exceptional in that it introduced changes to the module system that affected virtually every crate, even if those changes were almost completely automated. We expect that the changes introduced by most editions will be much more modest and discrete, more analogous to `async fn` (which simply introduced the `async` keyword), or the changes proposed by [RFC 2229] (which tweaks the way that closure captures work to make them more precise).
```

Comment:
"expect", while I also expect this, we may want to explain why we expect this and how we intend to maintain this expectation over time as the "we" changes.